Lib/test/test_enum.py

```diff
@@ -1776,19 +1776,12 @@ def _missing_(cls, item):
                     # trigger not found
                     return None
         self.assertIs(Color('three'), Color.blue)
-        self.assertRaises(ValueError, Color, 7)
-        try:
+        with self.assertRaises(ValueError):
+            Color(7)
+        with self.assertRaises(TypeError):
             Color('bad return')
-        except TypeError as exc:
-            self.assertTrue(isinstance(exc.__context__, ValueError))
```

Comment:
I think this test should probably be restored.

Reply:
Done